### PR TITLE
added style for simple modal

### PR DESCRIPTION
**Pull request by aishwarya-bm**
simple modal style with 2 action buttons

https://user-images.githubusercontent.com/66695068/152688245-69a17a4a-5621-4fae-9204-3c9f5269a49c.mp4

### CODE REVIEW

**Review by rahulcs754 — COMMENTED** [review]
all look good

File: stylesheets/components/modal.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  margin: 1rem 0;
+}
+
+.modal-action {
+  justify-content: flex-end;
+  margin-top: 0.5rem;
+}
+
+.modal h2 {
+  margin: 0 0 0.5rem 0;
```
Comment (vivekmehto) [inline stylesheets/components/modal.css]: You can use margin-bottom.

**Review by vivekmehto — COMMENTED** [review]
Looks nice just one suggestion you can use shorthand margin property.

File: stylesheets/components/modal.css (hunk 1 above)
Comment (aishwarya-bm) [reply stylesheets/components/modal.css]: Made the changes, thanks!

**Review by vivekmehto — APPROVED** [review]
Looks perfect.

**Pull request merged**